docs/FIREBASE_DEPLOY.md

```diff
@@ -0,0 +1,60 @@
+![firebase hosting](https://github.com/arundhati24/chat.susi.ai/blob/firebase_deploy/docs/img/blog_1_img.png)
```

Comment:
Same

Reply:
You need to change the link to point to FOSSASIA's repo. The broken image links will start to work when this PR gets merged :)